### PR TITLE
daemon: Add more info logs on startup

### PR DESCRIPTION
**Pull request by raybejjani**
Relates to: #3564

### CODE REVIEW

**Comment by raybejjani** [comment]
test-me-please

**Review by nebril — APPROVED** [review]

**Comment by raybejjani** [comment]
test-me-please

**Comment by raybejjani** [comment]
One thing for reviewers: should I add ellipsis to most of these messages? we do that on many "doing this thing..." logs we already have

**Comment by raybejjani** [comment]
test-me-please

File: daemon/daemon.go [hunk 1]
```diff
@@ -915,6 +915,7 @@ func (d *Daemon) init() error {
 
 // NewDaemon creates and returns a new Daemon with the parameters set in c.
 func NewDaemon(c *Config) (*Daemon, error) {
+	log.Info("Creating demon")
```
Comment (aanm) [inline daemon/daemon.go]: `s/demon/daemon`

**Review by aanm — CHANGES_REQUESTED** [review]
I would also add a `log.Info("Initialization completed, all systems go")` or something similar

File: daemon/daemon.go (hunk 1 above)
Comment (raybejjani) [reply daemon/daemon.go]: oops, that would have been funny

**Comment by raybejjani** [comment]
@aanm would an all systems go message be misleading? we've made a bunch of init stuff async so it might fail later. I'll add something to that effect but I'm worried that errors would show up after it.

**Comment by raybejjani** [comment]
test-me-please

File: daemon/main.go [hunk 2]
```diff
@@ -698,26 +698,33 @@ func initEnv(cmd *cobra.Command) {
...
 	if enableLogstash {
+		log.Info("Enabling Logstash")
 		go EnableLogstash(logstashAddr, int(logstashProbeTimer))
 	}
 
+	log.Info("Launching node monitor daemon")
 	go d.nodeMonitor.Run(path.Join(defaults.RuntimePath, defaults.EventsPipe))
 
 	// Launch cilium-health in the same namespace as cilium.
+	log.Info("Launching cilium health daemon")
```
Comment (rlenglet) [inline daemon/main.go]: cilium -> Cilium

**Review by rlenglet — APPROVED** [review]

File: daemon/main.go (hunk 2 above)
Comment (raybejjani) [reply daemon/main.go]: I assumed you also preferred the other lowercase ciliums switched, so I did. I hope that works!

**Comment by raybejjani** [comment]
test-me-please

**Review by tgraf — APPROVED** [review]

File: daemon/daemon.go (hunk 1 above)
Comment (tgraf) [reply daemon/daemon.go]: Tempted to keep it in

File: daemon/main.go (hunk 2 above)
Comment (rlenglet) [reply daemon/main.go]: Yes. Thank you! :)

**Review by aanm — APPROVED** [review]

**Pull request merged**